metadata.json

```diff
@@ -33,14 +33,12 @@
     {
       "operatingsystem": "CentOS",
       "operatingsystemrelease": [
-        "6",
```

Comment:
This should also be noted in the changelog. The best way to do that is to open an issue for it.

Reply:
since that's coming from https://github.com/voxpupuli/puppet-kafka/pull/322 I think we don't need a separate issue?

Reply:
Only if that's merged. If it's just closed it won't show up.

Reply:
Please let me know what comes out of this discussion.

Reply:
If you rebase this PR on top of #322, it'll automatically be marked as merged when this PR is merged. That's a strategy we've used multiple times and works best well.

Reply:
It is rebased on #322.